### PR TITLE
feat: added a way to restart services

### PR DESCRIPTION
**Pull request by h4ck3rk3y**
## Description:
This implements service restart - which is stop followed by start

## Is this change user facing?
YES

### CODE REVIEW

File: cli/cli/commands/service/restart/restart.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	starlarkScript = `
+def run(plan, args):
+	plan.stop_service(name=args["service_name"])
+	plan.start_service(name=args["service_name"])
+`
+)
+
+var ServiceRestartCmd = &engine_consuming_kurtosis_command.EngineConsumingKurtosisCommand{
+	CommandStr:                command_str_consts.ServiceRestartCmdStr,
+	ShortDescription:          "Restarts a service",
```
Comment (leoporoli) [inline cli/cli/commands/service/restart/restart.go]:
```suggestion
	ShortDescription:          "Restarts one or many services",
```

File: cli/cli/commands/service/restart/restart.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	LongDescription:           "Restarts service with the given service identifier in the given enclave",
```
Comment (leoporoli) [inline cli/cli/commands/service/restart/restart.go]:
```suggestion
	LongDescription:           "Restarts services with the given service identifiers in the given enclave",
```

File: cli/cli/commands/service/restart/restart.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+	args *args.ParsedArgs,
+) error {
+	enclaveIdentifier, err := args.GetNonGreedyArg(enclaveIdentifierArgKey)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred getting the enclave identifier value using key '%v'", enclaveIdentifierArgKey)
+	}
+
+	serviceIdentifiers, err := args.GetGreedyArg(serviceIdentifierArgKey)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred getting the service identifier value using key '%v'", serviceIdentifierArgKey)
```
Comment (leoporoli) [inline cli/cli/commands/service/restart/restart.go]:
```suggestion
		return stacktrace.Propagate(err, "An error occurred getting the service identifiers values using key '%v'", serviceIdentifierArgKey)
```

File: docs/docs/cli-reference/service-restart.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+slug: /service-restart
+---
+
+Services can be restarted like so:
+
+```bash
+kurtosis service restart $THE_ENCLAVE_IDENTIFIER $THE_SERVICE_IDENTIFIER
+```
+
+where `$THE_ENCLAVE_IDENTIFIER` and the `$THE_SERVICE_IDENTIFIER` are [resource identifiers](../advanced-concepts/resource-identifier.md) for the enclave and service, respectively.
```
Comment (leoporoli) [inline docs/docs/cli-reference/service-restart.md:13]: could we mention that a user can pass more than one service identifier?

**Review by leoporoli — APPROVED** [review]

File: docs/docs/cli-reference/service-restart.md (hunk 4 above)
Comment (h4ck3rk3y) [reply docs/docs/cli-reference/service-restart.md:13]: I will do that!

**Pull request closed without merging**